src/main/flight/mixer.c

```diff
@@ -512,8 +519,12 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
 
         const float rcCommandThrottleRange3dLow = rcCommand3dDeadBandLow - PWM_RANGE_MIN;
         const float rcCommandThrottleRange3dHigh = PWM_RANGE_MAX - rcCommand3dDeadBandHigh;
-
-        if (rcCommand[THROTTLE] <= rcCommand3dDeadBandLow || isFlipOverAfterCrashActive()) {
+        
+        if ((!mavlinkAttrateActive &&
```

Comment:
This should use the Betaflight RC protocol stack and not require custom code in `mixer.c`.